packages/remix-validated-form/src/ValidatedForm.tsx

```diff
@@ -364,18 +365,23 @@ export function ValidatedForm<DataType>({
     <Form
       ref={mergeRefs([formRef, formRefProp])}
       {...rest}
+      reloadDocument={reloadDocument}
       id={id}
       action={action}
       method={method}
       replace={replace}
-      onSubmit={(e) => {
-        e.preventDefault();
-        handleSubmit(
-          e,
-          e.currentTarget,
-          (e as unknown as HTMLSubmitEvent).nativeEvent
-        );
-      }}
+      onSubmit={
+        reloadDocument
```

Comment:
As-is, this is going to skip all validations, which I'm not sure is desirable. Shouldn't we validate, then submit the form via a full page reload? Not sure off the top of my head how to handle that situation when there's a `submitter` other than the form though.

Reply:
That’s a good point. I was going after this with the thought process that if you wanted a full page reload you would be okay with the server handling validation, like when JS isn’t running clientside. I can see how that’s not a super desirable solution though.

Reply:
Just hit this "bug" while upgrading from Remix 1.8, to 1.19.0.
Seems like something changed in the framework (probably when it was rewritten on top of react-router), that my use case fail:

`<ValidatedForm reloadDocument>` on a regular route, posting to a resource route that generates and download a PDF.

Used `patch-package` to implement this PR approach, but would definitely love something that keeps client-side validation.

Anything I can do to help this PR move forward?